beluga/include/beluga/sensor/data/regular_grid.hpp

```diff
@@ -130,6 +97,13 @@ class BaseRegularGrid2 : public ciabatta::ciabatta_top<Derived> {
     return cells | ranges::views::transform([this](const auto& cell) { return this->self().coordinates_at(cell); });
   }
 };
+/// Convenience alias for a 2D base regular grid.
+template <typename Derived>
+using BaseRegularGrid2 = BaseRegularGrid<Derived, 2>;
+
+/// Convenience alias for a 3D base regular grid.
+template <typename Derived>
+using BaseRegularGrid3 = BaseRegularGrid<Derived, 3>;
```

Comment:
@serraramiro1 consider partial specializations rather than type aliases to restore the handy overloads taking vector coordinates as scalar arguments. Same elsewhere.

Reply:
Mmmh I don't love the idea of providing different API's depending on the grid dimension.
I know there're alternatives with variadic templates, static assertions and Eigen::Map's, but that would need to be transfered to every single grid type we have, which IMO is overkill and yields little returns.

I'm inclined to leave this one as-is, unless you strongly feel otherwise?

Reply:
As per offline discussion, we will defer any "convenient" overloads to when we happen to need them.